scapy/main.py

```diff
@@ -31,9 +31,11 @@
 from scapy.themes import DefaultTheme, BlackAndWhite, apply_ipython_style
 from scapy.consts import WINDOWS
 
+from typing import cast, Any, Dict, List, Optional, Union
```

Comment:
The `six` module does not handle `typing`. I think that we can handle it in `scapy/compat.py`. If the typing module is installed, we return the real type. If not, we return a fake one.

Reply:
Good idea. It will cleaner and act as in http2.py